admin/app.js

```diff
@@ -32,7 +32,34 @@ const logger = require('./services/log.service').getLogger()
 const sqlService = require('./services/data-access/sql.service')
 const app = express()
 setupLogging(app)
-sqlService.initPool()
+
+/**
+ * Sleep in ms
+ * @param ms - milliseconds
+ * @return {Promise}
+ */
+function sleep (ms) {
+  return new Promise(resolve => {
+    setTimeout(resolve, ms)
+  })
+}
+
+// Initialise the Database Connection pool
```

Comment:
App.js is growing in size, may I suggest to isolate the init db conn pool in a separate file that is called from here?

Reply:
good point George, however i think this particular piece needs to be here to make a statement about its particular behaviour.  if anything, the route setup and other bigger chunks should be extracted